app/assets/javascripts/course/components/CoursePage.js

```diff
@@ -136,7 +152,15 @@ class CoursePage extends React.Component {
     } else if (!isLast(sections, section)) {
       this.displaySection(section.position + 1, 1, 1)
     } else {
-      // TODO: Congratulations Modal or if already completed, something else
+      const path = APIRoutes.finishedCoursePath(this.state.studentCourseId)
+      const updateParams = {
+        sent_email: true,
+      }
+      request.update(path, updateParams, (response) => {
+        console.log(response)
+      }, (error) => {
+        console.log(error)
+      })
     }
```

Comment:
I would put this block in its own function `updateFInishedCourse` or something similar